vcs/src/main/java/org/openjdk/skara/vcs/ReadOnlyRepository.java

```diff
@@ -66,6 +66,7 @@
         return files(h, Arrays.asList(paths));
     }
 
+    boolean dump(FileEntry entry, Path to) throws IOException;
```

Comment:
Is it useful to have a return value? Perhaps you are planning to use it in the future, but if so, it should probably have a test..